lib/gulp/css.js

```diff
@@ -28,12 +33,12 @@ function cssCompile(src, dest, minified, browserSync = false) {
   if (minified) {
     stream = stream
       .pipe(sass({outputStyle: 'compressed'}))
-      .pipe(postcss([rfs(), autoprefixer(), cssnano()]));
+      .pipe(postcss([rfs(), autoprefixer(), cssnano(), ...postCSSPlugins]));
```

Comment:
I would move the plugins before `autoprefixer` and `cssnano`, the plugins might output unprefixed or expanded CSS.